packages/components/psammead-bulletin/src/index.jsx

```diff
@@ -16,14 +20,100 @@ import { getPica, getLongPrimer } from '@bbc/gel-foundations/typography';
 import { mediaIcons } from '@bbc/psammead-assets/svgs';
 import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
 import { Link, LiveLabel } from '@bbc/psammead-story-promo';
+import { grid } from '@bbc/psammead-styles/detection';
+
+const twoOfSixColumnsMaxWidthScaleable = `33.33%`;
+// (2 / 6) * 100 = 0.3333333333 = 33.33%
+
+const fourOfSixColumnsMaxWidthScaleable = `66.67%`;
+// (4 / 6) * 100 = 66.6666666667 = 66.67%
+
+const fullWidthColumnsMaxScaleable = `100%`;
+// (12 / 12) * 100 = 100 = 100%
+
+const halfWidthColumnsMaxScaleable = `50%`;
+
+const bulletinWrapperStyles = `
+  position: relative;
+  background-color: ${C_LUNAR};
+  display: grid;
+  grid-template-columns: repeat(6, 1fr);
+  grid-column-gap: ${GEL_SPACING_DBL};
+`;
+
+const imageWrapperStyles = `
+  vertical-align: top;
+  display: inline-block;
+  grid-column: 1 / span 6;
+  @supports (${grid}) {
+    width: initial;
+  }
+`;
+
+const textWrapperStyles = `
+  grid-column: 1 / span 6;
+  display: inline-block;
+  width: ${fullWidthColumnsMaxScaleable};
+  @supports (${grid}) {
+    width: initial;
+    padding: 0;
+  }
+`;
+
+const TVBulletinWrapper = styled.div`
+  ${bulletinWrapperStyles};
+  padding: ${GEL_SPACING};
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    padding: ${GEL_SPACING_DBL};
+  }
+`;
+
+const TVImageWrapper = styled.div`
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    grid-column: 1 / span 3;
+    width: ${halfWidthColumnsMaxScaleable};
+  }
+  ${imageWrapperStyles};
```

Comment:
I feel like this should go at the start of the CSS, as this should hold the "defaults" which can then be overridden by the instance. What do you think?

Reply:
@AlistairGempf It should but having it on top means that a width of `50%` will be applied to the image. That is `50%` of the 3 columns it is supposed to span.

<img width="624" alt="Screenshot 2019-11-21 at 21 03 15" src="https://user-images.githubusercontent.com/48688870/69363972-7e372580-0ca2-11ea-9b3d-f4bce7b385cc.png">